android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -627,6 +630,24 @@ public void onReceivedError(
         new TopLoadingErrorEvent(webView.getId(), eventData));
     }
 
+    @Override
+    public void onReceivedHttpError(WebView webView, WebResourceRequest request, WebResourceResponse errorResponse) {
```

Comment:
Could this use `request.isForMainFrame()` to filter out errors from nested frames or include with the JS event whether or not it's for the main frame?